aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsCodegenDecorator.kt

```diff
@@ -63,6 +63,7 @@ val DECORATORS: List<ClientCodegenDecorator> =
             ServiceEnvConfigDecorator(),
             HttpRequestCompressionDecorator(),
         ),
+        RetryClassifierDecorator().applyExceptFor("com.amazonaws.s3#AmazonS3"),
```

Comment:
Add a comment on why S3 needs its own somewhere. Maybe just a link to this PR.

Reply:
Updated in b72a1fc